src/main.tsx

```diff
@@ -24,6 +24,8 @@ import { App } from "./App";
 import { init as initRageshake } from "./settings/rageshake";
 import { Initializer } from "./initializer";
 
+window.setLKLogLevel = setLKLogLevel;
```

Comment:
Can you stick this in a separate PR, please?

Reply:
I was wondering the same. So yes!